newsroom/factory/app.py

```diff
@@ -138,6 +141,7 @@ def setup_media_storage(self):
             self.media = AmazonMediaStorage(self)
```

Comment:
Until we add async S3 storage support, we can probably add `self.media_async = GridFSMediaStorageAsync()` here, otherwise if S3 is enabled `app.media_async` will be None